.nvmrc

```diff
@@ -1 +1 @@
-6.11.1
+8.3.0
```

Comment:
Bumping to 8.4.0. https://github.com/cerner/terra-core/pull/722/commits/8072a4cfec1e27b832b6f4a630a82a77f064c938 and https://github.com/cerner/terra-core/pull/722/commits/846f1d8fb93e5f8d9ea1a833f75f97d21ef4affb